packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/RemoteMessageSerializer.java

```diff
@@ -95,7 +95,7 @@ private static Bundle toBundle(@Nullable RemoteMessage.Notification notification
     serializedNotification.putStringArray("titleLocalizationArgs", notification.getTitleLocalizationArgs());
     serializedNotification.putString("titleLocalizationKey", notification.getTitleLocalizationKey());
     if (notification.getVibrateTimings() != null) {
-      serializedNotification.putIntArray("vibrateTimings", intArrayFromLongArray(notification.getVibrateTimings()));
+      serializedNotification.putLongArray("vibrateTimings", notification.getVibrateTimings());
```

Comment:
Glad to see that long arrays are now supported (they were not a couple of SDK versions ago)!